### PR TITLE
feature(crc): Sends submitted emails to parents

### PR DESCRIPTION
**Pull request by Rikkiabb**
# \<Description\>
Sends application submitted email to both parents when application gets sent to sýslumenn. 
Adds `AttachmentEmailTemplateGenerator` which takes email and file content. 
TODO: Implement design for email.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by VilliGunn — COMMENTED** [review]
Just some questions from me 💯

File: libs/application/template-api-modules/src/lib/modules/templates/children-residence-change/emailGenerators/applicationSubmitted.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+        Þú getur séð stöðu umsóknarinnar á mínum síðum á Ísland.is
+
+        <a href=${applicationLink} target="_blank">Fara á mínar síður</a>.
+      `
+
+  return {
+    from: {
+      name: 'Devland.is',
+      address: 'development@island.is',
```
[the remark this answers is not in the document]
Comment (Rikkiabb) [reply libs/application/template-api-modules/src/lib/modules/templates/children-residence-change/emailGenerators/applicationSubmitted.ts:30]: Agree, this is basically copy/paste from other email template generators in  template api. So hopefully @baering can help us with this question? 🙏

**Review by VilliGunn — APPROVED** [review]

File: libs/application/template-api-modules/src/lib/modules/templates/children-residence-change/emailGenerators/applicationSubmitted.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (baering) [reply libs/application/template-api-modules/src/lib/modules/templates/children-residence-change/emailGenerators/applicationSubmitted.ts:30]: Good point, just leave it as is for now and we will figure something out.

**Review by baering — APPROVED** [review]

**Pull request merged**